k8s/images/nginx/Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM byrnedo/alpine-curl
 
 # download all extra deps we need for the production container
-
 # templating executable
-RUN curl -L "https://github.com/gliderlabs/sigil/releases/download/v0.4.0/sigil_0.4.0_$(uname -sm|tr \  _).tgz" \ | tar -zxC /usr/bin
+COPY /k8s/images/nginx/download_sigil.sh /tmp/download_sigil.sh
+RUN chmod +x /tmp/download_sigil.sh
+RUN /tmp/download_sigil.sh
 
 FROM nginx:1.11
```

Comment:
Any reason to not update nginx while we're at it? I think 1.11 was EOL in 2017...

Reply:
Don't want to speak for Aron, but just to hold you over till he's out: I think he mentioned that he was running into some build issues after an upgrade? But I could be conflating nginx and the sigil work during our verbal updates.

Reply:
I can upgrade the version number, but I think it's best to do that in another PR. Keep the PRs small!

Reply:
That's the Sigil issue @DXCanas